utils/providers.py

```diff
@@ -49,7 +49,7 @@ def list_providers(allowed_types=None):
     return providers
 
 
-def get_mgmt(provider_key, providers=None, credentials=None):
+def get_mgmt(provider_key, providers=None, credentials=None, version=None):
```

Comment:
So we need to remove this in the prototype, it should get the version from the code below